homeassistant/components/lamarzocco/__init__.py

```diff
@@ -99,18 +105,26 @@ def bluetooth_configured() -> bool:
                 address_or_ble_device=entry.data[CONF_MAC],
             )
 
-    coordinator = LaMarzoccoUpdateCoordinator(
-        hass=hass,
-        entry=entry,
-        local_client=local_client,
+    device = LaMarzoccoMachine(
+        model=entry.data[CONF_MODEL],
+        serial_number=entry.unique_id,
+        name=entry.data[CONF_NAME],
         cloud_client=cloud_client,
+        local_client=local_client,
         bluetooth_client=bluetooth_client,
     )
 
-    await coordinator.async_config_entry_first_refresh()
-    entry.runtime_data = coordinator
+    coordinators = (
+        LaMarzoccoConfigUpdateCoordinator(hass, entry, device, local_client),
+        LaMarzoccoFirmwareUpdateCoordinator(hass, entry, device),
+        LaMarzoccoStatisticsUpdateCoordinator(hass, entry, device),
+    )
+    for coordinator in coordinators:
+        await coordinator.async_config_entry_first_refresh()
```

Comment:
Can we gather this and run these in parallel?

Reply:
nope, doesn't like that

Reply:
Let's add a comment for that in that case, as this is a thing that someone would instantly pick up to optimize (as this is a common pattern tools can find).

Reply:
done